MekHQ/build.gradle

```diff
@@ -140,26 +142,26 @@ task compileMML (type: GradleBuild) {
 }
 
 task mmlJar (type: Jar, dependsOn: compileMML) {
-    archiveName = "MegaMekLab.jar"
+    archiveFileName = "MegaMekLab.jar"
     from "${mmlDir}/build/classes/java/main"
```

Comment:
Do these need `megameklab`?

Reply:
All the mml source is in the top-level directory, unlike MM and MekHQ, which put everything in a subdirectory.